src/main/webapp/app/core/course/manage/course-management-container/course-management-container.component.ts

```diff
@@ -252,6 +253,8 @@ export class CourseManagementContainerComponent extends BaseCourseContainerCompo
             const childRouteComponent = this.activatedComponentReference() as CourseConversationsComponent;
             this.isSidebarCollapsed.set(childRouteComponent?.isCollapsed ?? false);
         }
+        // if we don't scroll to the top, the page will be scrolled to the last position which is not expected by the user
+        this.courseBody.nativeElement.scrollTop = 0;
```

Comment:
Why don't we use `utils#scrollToTopOfPage` here?
```suggestion
        scrollToTopOfPage;
```